html/browsers/windows/embedded-opener-remove-frame.html

```diff
@@ -15,32 +41,17 @@
     }
 
     // Test bits
-    assert_equals(frameW.opener, window, "opener before removal");
-
-    const openerDesc = Object.getOwnPropertyDescriptor(frameW, "opener"),
-          openerGet = openerDesc.get;
-
-    assert_equals(openerGet(), window, "opener before removal via directly invoking the getter");
-    frame.remove();
-    assert_equals(frameW.opener, null, "opener after removal");
-    assert_equals(openerGet(), null, "opener after removal via directly invoking the getter");
-
-    frameW.opener = null;
-    assert_equals(openerGet(), null, "opener after setting it null via directly invoking the getter");
-    const openerDescNull = Object.getOwnPropertyDescriptor(frameW, "opener");
-    assert_not_equals(openerDescNull, openerDesc);
-    assert_object_equals(openerDescNull, openerDesc);
-
-    frameW.opener = "immaterial";
-    assert_equals(openerGet(), null, "opener after setting it \"immaterial\" via directly invoking the getter");
-    const openerDescImmaterial = Object.getOwnPropertyDescriptor(frameW, "opener");
-    assert_equals(openerDescImmaterial.value, "immaterial");
-    assert_true(openerDescImmaterial.writable);
-    assert_true(openerDescImmaterial.enumerable);
-    assert_true(openerDescImmaterial.configurable);
+    testOpener(frameW, window, () => frame.remove());
 
     t.done();
```

Comment:
Can use t.step_func_done for this one too, right?

Reply:
No, due to different load event behavior.